enterprise/cmd/executor/internal/worker/workspace/unmount_windows.go

```
@@ -0,0 +1,7 @@
+package workspace
+
+import "errors"
+
+func unmount(dirPath string) error {
+	return errors.New("unmount not supported on Windows")
```

similarly, a panic here is probably appropriate since an error can be ignored more easily